src/tests/option_round/options_sold_tests.cairo

```diff
@@ -294,20 +316,18 @@ fn test_option_balance_per_bidder_after_auction_2() {
 #[available_gas(10000000)]
 fn test_option_round_options_sold_before_auction_end_is_0() {
     let (mut vault_facade, _) = setup_facade();
-    // LP deposits (into round 1)
-    let deposit_amount_wei: u256 = 10 * decimals();
-    vault_facade.deposit(deposit_amount_wei, liquidity_provider_1());
-    // Start auction
+
+    // Deposit liquidity and start the auction
     set_contract_address(vault_manager());
-    vault_facade.start_auction();
+    accelerate_to_auctioning(ref vault_facade);
+    // Make bids
     let mut current_round_facade: OptionRoundFacade = vault_facade.get_current_round();
+    let _params: OptionRoundParams = current_round_facade.get_params();
+
     // Make bid
     set_contract_address(option_bidder_buyer_1());
-    let option_params: OptionRoundParams = current_round_facade.get_params();
-    let bid_count: u256 = option_params.total_options_available + 10;
-    let bid_price: u256 = option_params.reserve_price;
-    let bid_amount: u256 = bid_count * bid_price;
-    current_round_facade.place_bid(bid_amount, bid_price, option_bidder_buyer_1());
+    accelerate_to_running(ref vault_facade);
```

Comment:
can't use accelerator here, we are testing that the options_sold is 0 before the auction ends, the accelerator ends it